ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -3592,6 +3592,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-nutanix-ipi-zones
+- as: nutanix-ipi-zones-multi-nic-f28
```

Comment:
I saw we already had `nutanix-ipi-zones-external-lb` and `nutanix-ipi-zones`, shall we use this one to replace `nutanix-ipi-zones` ?

Reply:
nutanix-ipi-zones-multi-nic is still a TP feature, maybe we can merge into nutanix-ipi-zones after GA, what do you think?

Reply:
ah, good point.